### PR TITLE
added voter documentation to readme

### PR DESCRIPTION
**Pull request by muhammedeminakbulut**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,25 @@ See the documentation for `zicht/cms` (https://github.com/zicht/cms)
...
+At least you should have these lines
+```
+security:
+    access_decision_manager:
+            strategy: unanimous
+```
+
+## PageVoter
+The PageVoter requires the page to have `Zicht\Bundle\PageBundle\Model\PageInterface` implemented.
+This Voter looks for the isPublic public function to check wether a page can be displayed to the public or not.
```
Comment (DoNormal) [inline README.md]: - Format isPublic with code markers: `isPublic`.
- "wether" should be spelled "whether".
- "whether" already implies "or not" so you can leave that out.

File: README.md [hunk 2]
```diff
@@ -20,7 +20,25 @@ See the documentation for `zicht/cms` (https://github.com/zicht/cms)
...
+
+## ScheduledContentVoter
+The ScheduledContentVoter requires the page to have `Zicht\Bundle\PageBundle\Model\ScheduledContentInterface` implemented.
+With this voter a page could be Scheduled for publication.
```
Comment (DoNormal) [inline README.md]: - The sentence sounds better when "could" is replaced with "can".
- "Scheduled" can be lowercase.

File: README.md [hunk 3]
```diff
@@ -20,7 +20,25 @@ See the documentation for `zicht/cms` (https://github.com/zicht/cms)
 for more information on how the `zicht/page-bundle` relates to the
 other Zicht Bundles
 
+## Voters
+To enable the use of [voters](http://symfony.com/doc/current/security/voters.html) in your configuration you should properly 
+configure `security.yml` or in your config the `security` part
```
Comment (DoNormal) [inline README.md]: - The part after "or" feels a bit wonky. Perhapse rewrite it to "or in the `security` part of your config." (Don't forget the dot at the end 😉 )

File: README.md [hunk 4]
```diff
@@ -20,7 +20,25 @@ See the documentation for `zicht/cms` (https://github.com/zicht/cms)
...
+
+At least you should have these lines
+```
+security:
+    access_decision_manager:
+            strategy: unanimous
+```
+
+## PageVoter
```
Comment (DoNormal) [inline README.md]: This is a part of the Voters section, which is marked up with two hashes, so this should be three hashes.

File: README.md [hunk 5]
```diff
@@ -20,7 +20,25 @@ See the documentation for `zicht/cms` (https://github.com/zicht/cms)
...
+The PageVoter requires the page to have `Zicht\Bundle\PageBundle\Model\PageInterface` implemented.
+This Voter looks for the isPublic public function to check wether a page can be displayed to the public or not.
+
+## ScheduledContentVoter
```
Comment (DoNormal) [inline README.md]: This is a part of the Voters section, which is marked up with two hashes, so this should be three hashes.

File: README.md [hunk 6]
```diff
@@ -20,7 +20,25 @@ See the documentation for `zicht/cms` (https://github.com/zicht/cms)
 for more information on how the `zicht/page-bundle` relates to the
 other Zicht Bundles
 
+## Voters
+To enable the use of [voters](http://symfony.com/doc/current/security/voters.html) in your configuration you should properly 
+configure `security.yml` or in your config the `security` part
+
+At least you should have these lines
```
Comment (DoNormal) [inline README.md]: Add a colon at the end?

**Review by TowardsDeath — CHANGES_REQUESTED** [review]

**Review by TowardsDeath — APPROVED** [review]

**Pull request merged**